configure.ac

```
@@ -267,10 +274,19 @@ AC_CHECK_FUNCS(fstatfs,[
 AC_CHECK_FUNCS(__lxstat,[
     LINK_WRAPPERS+=",-wrap,__lxstat"
 ],[])
+AC_CHECK_FUNCS(__lxstat64,[
+    LINK_WRAPPERS+=",-wrap,__lxstat64"
+],[])
 AC_CHECK_FUNCS(__xstat,[
     LINK_WRAPPERS+=",-wrap,__xstat"
 ],[])
+AC_CHECK_FUNCS(__xstat64,[
+    LINK_WRAPPERS+=",-wrap,__xstat64"
+],[])
 AC_CHECK_FUNCS(__fxstat,[
+    LINK_WRAPPERS+=",-wrap,__fxstat64"
+],[])
+AC_CHECK_FUNCS(__fxstat64,[
     LINK_WRAPPERS+=",-wrap,__fxstat"
 ],[])
```

A lot of parentheses and brackets so I could be seeing things here, but it looks like the `__fxstat` and `__fxstat64` are swapped here. I.e., the wrap for `__fxstat64` is inside the `__fxstat` function and vice-versa.

```suggestion
AC_CHECK_FUNCS(__fxstat,[
    LINK_WRAPPERS+=",-wrap,__fxstat"
],[])
AC_CHECK_FUNCS(__fxstat64,[
    LINK_WRAPPERS+=",-wrap,__fxstat64"
],[])
```

---

Good catch, must be something off during the squash, will fix it.